### ISSUE TITLE
High MSE Loss after 100 epoch (even higher than begaining)

### ISSUE THREAD

**Issue by JIAHELU** [EvelynFan/FaceFormer#105]
Hello author, thank you very much for your work. However, I encountered some issues during training. The MSE Loss fluctuates repeatedly during training, and it did not decrease to a relatively low level even by the 100th epoch. I feel like there might be some issues with the training details (such as learning rate, etc.). I wanted to ask, approximately what should the Loss be after training completion? Thank you!

![2024-03-26 11-43-56 的屏幕截图](https://github.com/EvelynFan/FaceFormer/assets/122361279/a5ad97b4-e6b4-40dd-abab-5bb9fcda5967)
